modules/core/test/test_utils.cpp

```diff
@@ -287,6 +290,53 @@ TEST(CommandLineParser, testScalar)
     EXPECT_EQ(parser.get<Scalar>("s5"), Scalar(5, -4, 3, 2));
 }
 
+
+TEST(Logger, message)
+{
+    int id = 42;
+    CV_LOG_VERBOSE(NULL, 0, "Verbose message: " << id);
+    CV_LOG_VERBOSE(NULL, 1, "Verbose message: " << id);
+    CV_LOG_DEBUG(NULL, "Debug message: " << id);
+    CV_LOG_INFO(NULL, "Info message: " << id);
```

Comment:
Google Test framework has 2 helper function to capture output: `testing::internal::CaptureStdout` and `testing::internal::CaptureStderr`. I would like to suggest to use the following pattern to check that message actually was logged:
```c++
    testing::internal::CaptureStdout();
    CV_LOG_INFO(NULL, "Info message: " << id);
    EXPECT_TRUE(testing::internal::GetCapturedStdout().find_first_of("Info message: 42") != std::string::npos);
```

Reply:
Thank you for the suggestion.

Actually I want to keep these tests as disabled for now - check compilation only (and run them in manual mode if we have corresponding changes).